src/extensions/extensionsApi.ts

```diff
@@ -458,14 +467,14 @@ export async function getExtensionVersion(extensionVersionRef: string): Promise<
       populateSpec(res.body.spec);
     }
     return res.body;
-  } catch (err: any) {
-    if (err.status === 404) {
+  } catch (err: unknown) {
+    if (isObject(err) && err.status === 404) {
```

Comment:
Should this just use getErrStatus?

Reply:
yup, I wrote this before I wrote that, and forgot to update it.